### PR TITLE
Feature/support multiversioning for content elements

### PR DESCRIPTION
**Pull request by hennejg**

### CODE REVIEW

File: neverpile-eureka-core/src/main/java/com/neverpile/eureka/rest/api/document/content/ContentElementResource.java [hunk 1]
```diff
@@ -336,16 +345,17 @@ public DocumentDto add(final HttpServletRequest request, @PathVariable("document
...
 
-  @PutMapping(value = "{documentID}/content/{content}", consumes = MediaType.MULTIPART_FORM_DATA_VALUE)
+  @PutMapping(value = "{documentID}/content/{content}", consumes = MediaType.ALL_VALUE)
   @Timed(description = "update content element", extraTags = {
       "operation", "update", "target", "content"
   }, value = "eureka.content.update")
-  public DocumentDto update(final HttpServletRequest request, @PathVariable("documentID") final String documentId,
-      @PathVariable("content") final String contentId, final AllRequestParts files, // mapped using
-                                                                                    // AllRequestPartsMethodArgumentResolver
+  public ContentElement update(final HttpServletRequest request, @PathVariable("documentID") final String documentId,
```
Comment (hennejg) [inline neverpile-eureka-core/src/main/java/com/neverpile/eureka/rest/api/document/content/ContentElementResource.java:355]: This is a breaking change: the former semantics, i.e. replace a single CE with a multipart upload and return the document, didn't make a lot of sense. Therefore the new semantics is that a single element can be updated, returning the new descriptor. 

We may want to allow multipart update in the future but then under `...documents/{documentId}/content` and not under `...documents/{documentId}/content/{contentId}`.

Comment (Crow08) [reply neverpile-eureka-core/src/main/java/com/neverpile/eureka/rest/api/document/content/ContentElementResource.java:359]: makes sense!

File: neverpile-eureka-core/src/main/java/com/neverpile/eureka/rest/api/document/content/ContentElementResource.java [hunk 2]
```diff
@@ -336,16 +349,17 @@ public DocumentDto add(final HttpServletRequest request, @PathVariable("document
...
+      @PathVariable("content") final String contentId, final //
```
Comment (Crow08) [inline neverpile-eureka-core/src/main/java/com/neverpile/eureka/rest/api/document/content/ContentElementResource.java:360]: nitpick: weird linebreak

Comment (hennejg) [reply neverpile-eureka-core/src/main/java/com/neverpile/eureka/rest/api/document/content/ContentElementResource.java:360]: Done

**Pull request merged**